utils/pythonutils/poetryutils.go

```diff
@@ -0,0 +1,134 @@
+package pythonutils
+
+import (
+	"errors"
+	"os"
+
+	"github.com/BurntSushi/toml"
+)
+
+type PyprojectToml struct {
+	Tool map[string]PoetryPackage
+}
+type PoetryPackage struct {
+	Name            string
+	Version         string
+	Dependencies    map[string]interface{}
+	DevDependencies map[string]interface{} `toml:"dev-dependencies"`
+}
+
+type PoetryLock struct {
+	Package []*PoetryPackage
+}
+
+// Executes the poetry-dependency-map script and returns a dependency map of all the installed poetry packages in the current environment and another list of the top level dependencies
```

Comment:
Let's better describe the structure of the returned values. Maybe add an examples to help the reader understand the structure.